From d5c9e73f36c946e403d408b08d7360f227c75837 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Tue, 14 May 2019 13:33:05 +0100
Subject: [PATCH 504/725] overlays: Standardise on compatible="brcm,bcm2835"

Curb the proliferation of compatible string combinations by
standardising on "brcm,bcm2835" to denote BCM2835 and its descendants.

As nothing in the firmware or kernel is checking overlay compatible
strings, this should be a purely cosmetic change.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/overlays/adau1977-adc-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/adau7002-simple-overlay.dts          | 2 +-
 arch/arm/boot/dts/overlays/ads1015-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/ads1115-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/ads7846-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/adv7282m-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/adv728x-m-overlay.dts                | 2 +-
 arch/arm/boot/dts/overlays/akkordion-iqdacplus-overlay.dts      | 2 +-
 .../boot/dts/overlays/allo-boss-dac-pcm512x-audio-overlay.dts   | 2 +-
 arch/arm/boot/dts/overlays/allo-digione-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/allo-katana-dac-audio-overlay.dts    | 2 +-
 .../boot/dts/overlays/allo-piano-dac-pcm512x-audio-overlay.dts  | 2 +-
 .../dts/overlays/allo-piano-dac-plus-pcm512x-audio-overlay.dts  | 2 +-
 arch/arm/boot/dts/overlays/applepi-dac-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/at86rf233-overlay.dts                | 2 +-
 arch/arm/boot/dts/overlays/audioinjector-addons-overlay.dts     | 2 +-
 arch/arm/boot/dts/overlays/audioinjector-ultra-overlay.dts      | 2 +-
 .../boot/dts/overlays/audioinjector-wm8731-audio-overlay.dts    | 2 +-
 arch/arm/boot/dts/overlays/audiosense-pi-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/audremap-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/balena-fin-overlay.dts               | 2 +-
 arch/arm/boot/dts/overlays/bmp085_i2c-sensor-overlay.dts        | 2 +-
 arch/arm/boot/dts/overlays/dht11-overlay.dts                    | 2 +-
 arch/arm/boot/dts/overlays/dionaudio-loco-overlay.dts           | 2 +-
 arch/arm/boot/dts/overlays/dionaudio-loco-v2-overlay.dts        | 2 +-
 arch/arm/boot/dts/overlays/dpi18-overlay.dts                    | 2 +-
 arch/arm/boot/dts/overlays/dpi24-overlay.dts                    | 2 +-
 arch/arm/boot/dts/overlays/draws-overlay.dts                    | 2 +-
 arch/arm/boot/dts/overlays/dwc-otg-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/dwc2-overlay.dts                     | 2 +-
 arch/arm/boot/dts/overlays/enc28j60-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/enc28j60-spi2-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/exc3000-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/goodix-overlay.dts                   | 2 +-
 arch/arm/boot/dts/overlays/googlevoicehat-soundcard-overlay.dts | 2 +-
 arch/arm/boot/dts/overlays/gpio-fan-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/gpio-ir-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/gpio-ir-tx-overlay.dts               | 2 +-
 arch/arm/boot/dts/overlays/gpio-key-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/gpio-poweroff-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/gpio-shutdown-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/hifiberry-amp-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/hifiberry-dac-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/hifiberry-dacplus-overlay.dts        | 2 +-
 arch/arm/boot/dts/overlays/hifiberry-dacplusadc-overlay.dts     | 2 +-
 arch/arm/boot/dts/overlays/hifiberry-digi-overlay.dts           | 2 +-
 arch/arm/boot/dts/overlays/hifiberry-digi-pro-overlay.dts       | 2 +-
 arch/arm/boot/dts/overlays/hy28a-overlay.dts                    | 2 +-
 arch/arm/boot/dts/overlays/hy28b-2017-overlay.dts               | 2 +-
 arch/arm/boot/dts/overlays/hy28b-overlay.dts                    | 2 +-
 arch/arm/boot/dts/overlays/i-sabre-q2m-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/i2c-bcm2708-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/i2c-mux-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/i2c-pwm-pca9685a-overlay.dts         | 2 +-
 arch/arm/boot/dts/overlays/i2c-rtc-gpio-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/i2c-rtc-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/i2c-sensor-overlay.dts               | 2 +-
 arch/arm/boot/dts/overlays/i2c0-bcm2708-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/i2c1-bcm2708-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/i2s-gpio28-31-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/ilitek251x-overlay.dts               | 2 +-
 arch/arm/boot/dts/overlays/iqaudio-dac-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/iqaudio-dacplus-overlay.dts          | 2 +-
 .../arm/boot/dts/overlays/iqaudio-digi-wm8804-audio-overlay.dts | 2 +-
 arch/arm/boot/dts/overlays/jedec-spi-nor-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/justboom-dac-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/justboom-digi-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/max98357a-overlay.dts                | 2 +-
 arch/arm/boot/dts/overlays/mbed-dac-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/mcp23017-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/mcp23s17-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/mcp2515-can0-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/mcp2515-can1-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/mcp3008-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/mcp3202-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/mcp342x-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/media-center-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/mmc-overlay.dts                      | 2 +-
 arch/arm/boot/dts/overlays/mpu6050-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/mz61581-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/ov5647-overlay.dts                   | 2 +-
 arch/arm/boot/dts/overlays/papirus-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/pi3-act-led-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/pi3-disable-bt-overlay.dts           | 2 +-
 arch/arm/boot/dts/overlays/pi3-disable-wifi-overlay.dts         | 2 +-
 arch/arm/boot/dts/overlays/pi3-miniuart-bt-overlay.dts          | 2 +-
 arch/arm/boot/dts/overlays/pibell-overlay.dts                   | 2 +-
 arch/arm/boot/dts/overlays/piglow-overlay.dts                   | 2 +-
 arch/arm/boot/dts/overlays/piscreen-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/piscreen2r-overlay.dts               | 2 +-
 arch/arm/boot/dts/overlays/pisound-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/pitft22-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/pitft28-capacitive-overlay.dts       | 2 +-
 arch/arm/boot/dts/overlays/pitft28-resistive-overlay.dts        | 2 +-
 arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts        | 2 +-
 arch/arm/boot/dts/overlays/pps-gpio-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/pwm-ir-tx-overlay.dts                | 2 +-
 arch/arm/boot/dts/overlays/qca7000-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/rotary-encoder-overlay.dts           | 2 +-
 arch/arm/boot/dts/overlays/rpi-backlight-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/rpi-cirrus-wm5102-overlay.dts        | 2 +-
 arch/arm/boot/dts/overlays/rpi-dac-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/rpi-display-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts               | 2 +-
 arch/arm/boot/dts/overlays/rpi-poe-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/rpi-proto-overlay.dts                | 2 +-
 arch/arm/boot/dts/overlays/rpi-sense-overlay.dts                | 2 +-
 arch/arm/boot/dts/overlays/rpi-tv-overlay.dts                   | 2 +-
 .../arm/boot/dts/overlays/rra-digidac1-wm8741-audio-overlay.dts | 2 +-
 arch/arm/boot/dts/overlays/sc16is750-i2c-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/sc16is752-spi1-overlay.dts           | 2 +-
 arch/arm/boot/dts/overlays/sdhost-overlay.dts                   | 2 +-
 arch/arm/boot/dts/overlays/sdio-overlay.dts                     | 2 +-
 arch/arm/boot/dts/overlays/sdtweak-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/smi-nand-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/smi-overlay.dts                      | 2 +-
 arch/arm/boot/dts/overlays/spi-gpio35-39-overlay.dts            | 2 +-
 arch/arm/boot/dts/overlays/spi-rtc-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/spi0-cs-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/spi0-hw-cs-overlay.dts               | 2 +-
 arch/arm/boot/dts/overlays/spi1-1cs-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/spi1-2cs-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/spi1-3cs-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/spi2-1cs-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/spi2-2cs-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/spi2-3cs-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/ssd1306-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/superaudioboard-overlay.dts          | 2 +-
 arch/arm/boot/dts/overlays/sx150x-overlay.dts                   | 2 +-
 arch/arm/boot/dts/overlays/tc358743-audio-overlay.dts           | 2 +-
 arch/arm/boot/dts/overlays/tc358743-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/tinylcd35-overlay.dts                | 2 +-
 arch/arm/boot/dts/overlays/tpm-slb9670-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/uart0-overlay.dts                    | 2 +-
 arch/arm/boot/dts/overlays/uart1-overlay.dts                    | 2 +-
 arch/arm/boot/dts/overlays/udrc-overlay.dts                     | 2 +-
 arch/arm/boot/dts/overlays/upstream-overlay.dts                 | 2 +-
 arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts             | 2 +-
 arch/arm/boot/dts/overlays/vc4-kms-kippah-7inch-overlay.dts     | 2 +-
 arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts              | 2 +-
 arch/arm/boot/dts/overlays/vga666-overlay.dts                   | 2 +-
 arch/arm/boot/dts/overlays/w1-gpio-overlay.dts                  | 2 +-
 arch/arm/boot/dts/overlays/w1-gpio-pullup-overlay.dts           | 2 +-
 arch/arm/boot/dts/overlays/wittypi-overlay.dts                  | 2 +-
 146 files changed, 146 insertions(+), 146 deletions(-)

--- a/arch/arm/boot/dts/overlays/adau1977-adc-overlay.dts
+++ b/arch/arm/boot/dts/overlays/adau1977-adc-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
     
 	fragment@0 {
         	target = <&i2c>;
--- a/arch/arm/boot/dts/overlays/adau7002-simple-overlay.dts
+++ b/arch/arm/boot/dts/overlays/adau7002-simple-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2708";
+    compatible = "brcm,bcm2835";
 
     fragment@0 {
         target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/ads1015-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ads1015-overlay.dts
@@ -5,7 +5,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2708";
+    compatible = "brcm,bcm2835";
     /* ----------- ADS1015 ------------ */
     fragment@0 {
         target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/ads1115-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ads1115-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/ads7846-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ads7846-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/adv7282m-overlay.dts
+++ b/arch/arm/boot/dts/overlays/adv7282m-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_vc>;
--- a/arch/arm/boot/dts/overlays/adv728x-m-overlay.dts
+++ b/arch/arm/boot/dts/overlays/adv728x-m-overlay.dts
@@ -5,7 +5,7 @@
 #include "adv7282m-overlay.dts"
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	// Fragment numbers deliberately high to avoid conflicts with the
 	// included adv7282m overlay file.
--- a/arch/arm/boot/dts/overlays/akkordion-iqdacplus-overlay.dts
+++ b/arch/arm/boot/dts/overlays/akkordion-iqdacplus-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/allo-boss-dac-pcm512x-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/allo-boss-dac-pcm512x-audio-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/clocks";
--- a/arch/arm/boot/dts/overlays/allo-digione-overlay.dts
+++ b/arch/arm/boot/dts/overlays/allo-digione-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/allo-katana-dac-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/allo-katana-dac-audio-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/allo-piano-dac-pcm512x-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/allo-piano-dac-pcm512x-audio-overlay.dts
@@ -13,7 +13,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/allo-piano-dac-plus-pcm512x-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/allo-piano-dac-plus-pcm512x-audio-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/applepi-dac-overlay.dts
+++ b/arch/arm/boot/dts/overlays/applepi-dac-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2708";
+    compatible = "brcm,bcm2835";
 
     fragment@0 {
         target = <&sound>;
--- a/arch/arm/boot/dts/overlays/at86rf233-overlay.dts
+++ b/arch/arm/boot/dts/overlays/at86rf233-overlay.dts
@@ -4,7 +4,7 @@
 /* Overlay for Atmel AT86RF233 IEEE 802.15.4 WPAN transceiver on spi0.0 */
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2836", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/audioinjector-addons-overlay.dts
+++ b/arch/arm/boot/dts/overlays/audioinjector-addons-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/audioinjector-ultra-overlay.dts
+++ b/arch/arm/boot/dts/overlays/audioinjector-ultra-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/audioinjector-wm8731-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/audioinjector-wm8731-audio-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/audiosense-pi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/audiosense-pi-overlay.dts
@@ -5,7 +5,7 @@
 #include <dt-bindings/gpio/gpio.h>
 
 / {
-	compatible = "brcm,bcm2837", "brcm,bcm2836", "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/audremap-overlay.dts
+++ b/arch/arm/boot/dts/overlays/audremap-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-        compatible = "brcm,bcm2708";
+        compatible = "brcm,bcm2835";
 
         fragment@0 {
                 target = <&audio_pins>;
--- a/arch/arm/boot/dts/overlays/balena-fin-overlay.dts
+++ b/arch/arm/boot/dts/overlays/balena-fin-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&mmc>;
--- a/arch/arm/boot/dts/overlays/bmp085_i2c-sensor-overlay.dts
+++ b/arch/arm/boot/dts/overlays/bmp085_i2c-sensor-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-        compatible = "brcm,bcm2708";
+        compatible = "brcm,bcm2835";
 
         fragment@0 {
                 target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/dht11-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dht11-overlay.dts
@@ -5,7 +5,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/dionaudio-loco-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dionaudio-loco-overlay.dts
@@ -8,7 +8,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/dionaudio-loco-v2-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dionaudio-loco-v2-overlay.dts
@@ -9,7 +9,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&sound>;
--- a/arch/arm/boot/dts/overlays/dpi18-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dpi18-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	// There is no DPI driver module, but we need a platform device
 	// node (that doesn't already use pinctrl) to hang the pinctrl
--- a/arch/arm/boot/dts/overlays/dpi24-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dpi24-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	// There is no DPI driver module, but we need a platform device
 	// node (that doesn't already use pinctrl) to hang the pinctrl
--- a/arch/arm/boot/dts/overlays/draws-overlay.dts
+++ b/arch/arm/boot/dts/overlays/draws-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2835", "brcm,bcm2836", "brcm,bcm2708", "brcm,bcm2709";
+    compatible = "brcm,bcm2835";
     fragment@0 {
         target = <&i2s>;
         __overlay__ {
--- a/arch/arm/boot/dts/overlays/dwc-otg-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dwc-otg-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&usb>;
--- a/arch/arm/boot/dts/overlays/dwc2-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dwc2-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&usb>;
--- a/arch/arm/boot/dts/overlays/enc28j60-overlay.dts
+++ b/arch/arm/boot/dts/overlays/enc28j60-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/enc28j60-spi2-overlay.dts
+++ b/arch/arm/boot/dts/overlays/enc28j60-spi2-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi2>;
--- a/arch/arm/boot/dts/overlays/exc3000-overlay.dts
+++ b/arch/arm/boot/dts/overlays/exc3000-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&clocks>;
--- a/arch/arm/boot/dts/overlays/goodix-overlay.dts
+++ b/arch/arm/boot/dts/overlays/goodix-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/googlevoicehat-soundcard-overlay.dts
+++ b/arch/arm/boot/dts/overlays/googlevoicehat-soundcard-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/gpio-fan-overlay.dts
+++ b/arch/arm/boot/dts/overlays/gpio-fan-overlay.dts
@@ -38,7 +38,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/gpio-ir-overlay.dts
+++ b/arch/arm/boot/dts/overlays/gpio-ir-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-        compatible = "brcm,bcm2708";
+        compatible = "brcm,bcm2835";
 
         fragment@0 {
                 target-path = "/";
--- a/arch/arm/boot/dts/overlays/gpio-ir-tx-overlay.dts
+++ b/arch/arm/boot/dts/overlays/gpio-ir-tx-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/gpio-key-overlay.dts
+++ b/arch/arm/boot/dts/overlays/gpio-key-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		// Configure the gpio pin controller
--- a/arch/arm/boot/dts/overlays/gpio-poweroff-overlay.dts
+++ b/arch/arm/boot/dts/overlays/gpio-poweroff-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/gpio-shutdown-overlay.dts
+++ b/arch/arm/boot/dts/overlays/gpio-shutdown-overlay.dts
@@ -10,7 +10,7 @@
 // note that GPIO3 has an external pullup on at least some boards).
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		// Configure the gpio pin controller
--- a/arch/arm/boot/dts/overlays/hifiberry-amp-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hifiberry-amp-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/hifiberry-dac-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hifiberry-dac-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/hifiberry-dacplus-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hifiberry-dacplus-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/clocks";
--- a/arch/arm/boot/dts/overlays/hifiberry-dacplusadc-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hifiberry-dacplusadc-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/clocks";
--- a/arch/arm/boot/dts/overlays/hifiberry-digi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hifiberry-digi-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/hifiberry-digi-pro-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hifiberry-digi-pro-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/hy28a-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hy28a-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/hy28b-2017-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hy28b-2017-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/hy28b-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hy28b-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/i-sabre-q2m-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i-sabre-q2m-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&sound>;
--- a/arch/arm/boot/dts/overlays/i2c-bcm2708-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-bcm2708-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/i2c-mux-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-mux-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/i2c-pwm-pca9685a-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-pwm-pca9685a-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/i2c-rtc-gpio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-rtc-gpio-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/i2c-rtc-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-rtc-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/i2c-sensor-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-sensor-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/i2c0-bcm2708-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c0-bcm2708-overlay.dts
@@ -9,7 +9,7 @@
 /plugin/;
 
 /{
-   compatible = "brcm,bcm2708";
+   compatible = "brcm,bcm2835";
 
    fragment@0 {
       target = <&i2c0>;
--- a/arch/arm/boot/dts/overlays/i2c1-bcm2708-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c1-bcm2708-overlay.dts
@@ -9,7 +9,7 @@
 /plugin/;
 
 /{
-   compatible = "brcm,bcm2708";
+   compatible = "brcm,bcm2835";
 
    fragment@0 {
       target = <&i2c1>;
--- a/arch/arm/boot/dts/overlays/i2s-gpio28-31-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2s-gpio28-31-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2836", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s_pins>;
--- a/arch/arm/boot/dts/overlays/ilitek251x-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ilitek251x-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
  / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
  	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/iqaudio-dac-overlay.dts
+++ b/arch/arm/boot/dts/overlays/iqaudio-dac-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/iqaudio-dacplus-overlay.dts
+++ b/arch/arm/boot/dts/overlays/iqaudio-dacplus-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/iqaudio-digi-wm8804-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/iqaudio-digi-wm8804-audio-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/jedec-spi-nor-overlay.dts
+++ b/arch/arm/boot/dts/overlays/jedec-spi-nor-overlay.dts
@@ -13,7 +13,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	// disable spi-dev on spi0.0
 	fragment@0 {
--- a/arch/arm/boot/dts/overlays/justboom-dac-overlay.dts
+++ b/arch/arm/boot/dts/overlays/justboom-dac-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/justboom-digi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/justboom-digi-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/max98357a-overlay.dts
+++ b/arch/arm/boot/dts/overlays/max98357a-overlay.dts
@@ -8,7 +8,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	/* Enable I2S */
 	fragment@0 {
--- a/arch/arm/boot/dts/overlays/mbed-dac-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mbed-dac-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/mcp23017-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mcp23017-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c1>;
--- a/arch/arm/boot/dts/overlays/mcp23s17-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mcp23s17-overlay.dts
@@ -20,7 +20,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	// disable spi-dev on spi0.0
 	fragment@0 {
--- a/arch/arm/boot/dts/overlays/mcp2515-can0-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mcp2515-can0-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2835", "brcm,bcm2836", "brcm,bcm2708", "brcm,bcm2709";
+    compatible = "brcm,bcm2835";
     /* disable spi-dev for spi0.0 */
     fragment@0 {
         target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/mcp2515-can1-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mcp2515-can1-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2835", "brcm,bcm2836", "brcm,bcm2708", "brcm,bcm2709";
+    compatible = "brcm,bcm2835";
     /* disable spi-dev for spi0.1 */
     fragment@0 {
         target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/mcp3008-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mcp3008-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spidev0>;
--- a/arch/arm/boot/dts/overlays/mcp3202-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mcp3202-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spidev0>;
--- a/arch/arm/boot/dts/overlays/mcp342x-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mcp342x-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c1>;
--- a/arch/arm/boot/dts/overlays/media-center-overlay.dts
+++ b/arch/arm/boot/dts/overlays/media-center-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/mmc-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mmc-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&mmc>;
--- a/arch/arm/boot/dts/overlays/mpu6050-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mpu6050-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-        compatible = "brcm,bcm2708";
+        compatible = "brcm,bcm2835";
 
         fragment@0 {
                 target = <&i2c1>;
--- a/arch/arm/boot/dts/overlays/mz61581-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mz61581-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/ov5647-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ov5647-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_vc>;
--- a/arch/arm/boot/dts/overlays/papirus-overlay.dts
+++ b/arch/arm/boot/dts/overlays/papirus-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/pi3-act-led-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pi3-act-led-overlay.dts
@@ -11,7 +11,7 @@
 */
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&act_led>;
--- a/arch/arm/boot/dts/overlays/pi3-disable-bt-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pi3-disable-bt-overlay.dts
@@ -9,7 +9,7 @@
 */
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&uart1>;
--- a/arch/arm/boot/dts/overlays/pi3-disable-wifi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pi3-disable-wifi-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&mmc>;
--- a/arch/arm/boot/dts/overlays/pi3-miniuart-bt-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pi3-miniuart-bt-overlay.dts
@@ -16,7 +16,7 @@
 */
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&uart0>;
--- a/arch/arm/boot/dts/overlays/pibell-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pibell-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2708";
+    compatible = "brcm,bcm2835";
 
     fragment@0 {
         target-path = "/";
--- a/arch/arm/boot/dts/overlays/piglow-overlay.dts
+++ b/arch/arm/boot/dts/overlays/piglow-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/piscreen-overlay.dts
+++ b/arch/arm/boot/dts/overlays/piscreen-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/piscreen2r-overlay.dts
+++ b/arch/arm/boot/dts/overlays/piscreen2r-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/pisound-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pisound-overlay.dts
@@ -23,7 +23,7 @@
 #include <dt-bindings/gpio/gpio.h>
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/pitft22-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pitft22-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-        compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+        compatible = "brcm,bcm2835";
 
         fragment@0 {
                 target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/pitft28-capacitive-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pitft28-capacitive-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-        compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+        compatible = "brcm,bcm2835";
 
         fragment@0 {
                 target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/pitft28-resistive-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pitft28-resistive-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/pps-gpio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pps-gpio-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 	fragment@0 {
 		target-path = "/";
 		__overlay__ {
--- a/arch/arm/boot/dts/overlays/pwm-ir-tx-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pwm-ir-tx-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/qca7000-overlay.dts
+++ b/arch/arm/boot/dts/overlays/qca7000-overlay.dts
@@ -5,7 +5,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spidev0>;
--- a/arch/arm/boot/dts/overlays/rotary-encoder-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rotary-encoder-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/rpi-backlight-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-backlight-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/rpi-cirrus-wm5102-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-cirrus-wm5102-overlay.dts
@@ -6,7 +6,7 @@
 #include <dt-bindings/mfd/arizona.h>
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/rpi-dac-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-dac-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/rpi-display-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-display-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/rpi-poe-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-poe-overlay.dts
@@ -5,7 +5,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/rpi-proto-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-proto-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/rpi-sense-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-sense-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c1>;
--- a/arch/arm/boot/dts/overlays/rpi-tv-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-tv-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spidev0>;
--- a/arch/arm/boot/dts/overlays/rra-digidac1-wm8741-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rra-digidac1-wm8741-audio-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/sc16is750-i2c-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sc16is750-i2c-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_arm>;
--- a/arch/arm/boot/dts/overlays/sc16is752-spi1-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sc16is752-spi1-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/sdhost-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sdhost-overlay.dts
@@ -4,7 +4,7 @@
 /* Provide backwards compatible aliases for the old sdhost dtparams. */
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&sdhost>;
--- a/arch/arm/boot/dts/overlays/sdio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sdio-overlay.dts
@@ -4,7 +4,7 @@
 /* Enable SDIO from MMC interface via GPIOs 22-27. Includes sdhost overlay. */
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&mmc>;
--- a/arch/arm/boot/dts/overlays/sdtweak-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sdtweak-overlay.dts
@@ -4,7 +4,7 @@
 /* Provide backwards compatible aliases for the old sdhost dtparams. */
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&sdhost>;
--- a/arch/arm/boot/dts/overlays/smi-nand-overlay.dts
+++ b/arch/arm/boot/dts/overlays/smi-nand-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&smi>;
--- a/arch/arm/boot/dts/overlays/smi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/smi-overlay.dts
@@ -5,7 +5,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&smi>;
--- a/arch/arm/boot/dts/overlays/spi-gpio35-39-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi-gpio35-39-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2836", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/spi-rtc-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi-rtc-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spidev0>;
--- a/arch/arm/boot/dts/overlays/spi0-cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi0-cs-overlay.dts
@@ -3,7 +3,7 @@
 
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0_cs_pins>;
--- a/arch/arm/boot/dts/overlays/spi0-hw-cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi0-hw-cs-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2836", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/spi1-1cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi1-1cs-overlay.dts
@@ -3,7 +3,7 @@
 
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/spi1-2cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi1-2cs-overlay.dts
@@ -3,7 +3,7 @@
 
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/spi1-3cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi1-3cs-overlay.dts
@@ -3,7 +3,7 @@
 
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/spi2-1cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi2-1cs-overlay.dts
@@ -3,7 +3,7 @@
 
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/spi2-2cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi2-2cs-overlay.dts
@@ -3,7 +3,7 @@
 
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/spi2-3cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi2-3cs-overlay.dts
@@ -3,7 +3,7 @@
 
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&gpio>;
--- a/arch/arm/boot/dts/overlays/ssd1306-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ssd1306-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2718";
+    compatible = "brcm,bcm2835";
 
     fragment@0 {
 	target = <&i2c1>;
--- a/arch/arm/boot/dts/overlays/superaudioboard-overlay.dts
+++ b/arch/arm/boot/dts/overlays/superaudioboard-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&sound>;
--- a/arch/arm/boot/dts/overlays/sx150x-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sx150x-overlay.dts
@@ -22,7 +22,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	// Enable I2C#0 interface
 	fragment@0 {
--- a/arch/arm/boot/dts/overlays/tc358743-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/tc358743-audio-overlay.dts
@@ -5,7 +5,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
--- a/arch/arm/boot/dts/overlays/tc358743-overlay.dts
+++ b/arch/arm/boot/dts/overlays/tc358743-overlay.dts
@@ -4,7 +4,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2c_vc>;
--- a/arch/arm/boot/dts/overlays/tinylcd35-overlay.dts
+++ b/arch/arm/boot/dts/overlays/tinylcd35-overlay.dts
@@ -24,7 +24,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/tpm-slb9670-overlay.dts
+++ b/arch/arm/boot/dts/overlays/tpm-slb9670-overlay.dts
@@ -8,7 +8,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&spi0>;
--- a/arch/arm/boot/dts/overlays/uart0-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart0-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&uart0>;
--- a/arch/arm/boot/dts/overlays/uart1-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart1-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&uart1>;
--- a/arch/arm/boot/dts/overlays/udrc-overlay.dts
+++ b/arch/arm/boot/dts/overlays/udrc-overlay.dts
@@ -7,7 +7,7 @@
 /plugin/;
 
 / {
-    compatible = "brcm,bcm2835", "brcm,bcm2836", "brcm,bcm2708", "brcm,bcm2709";
+    compatible = "brcm,bcm2835";
     fragment@0 {
         target = <&i2s>;
         __overlay__ {
--- a/arch/arm/boot/dts/overlays/upstream-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-overlay.dts
@@ -6,7 +6,7 @@
 #include <dt-bindings/clock/bcm2835.h>
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 	fragment@0 {
 		target-path = "/chosen";
 		__dormant__ {
--- a/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-fkms-v3d-overlay.dts
@@ -6,7 +6,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/chosen";
--- a/arch/arm/boot/dts/overlays/vc4-kms-kippah-7inch-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-kippah-7inch-overlay.dts
@@ -8,7 +8,7 @@
 #include <dt-bindings/pinctrl/bcm2835.h>
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
@@ -8,7 +8,7 @@
 #include <dt-bindings/clock/bcm2835.h>
 
 / {
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/chosen";
--- a/arch/arm/boot/dts/overlays/vga666-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vga666-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 /{
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	// There is no VGA driver module, but we need a platform device
 	// node (that doesn't already use pinctrl) to hang the pinctrl
--- a/arch/arm/boot/dts/overlays/w1-gpio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/w1-gpio-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/w1-gpio-pullup-overlay.dts
+++ b/arch/arm/boot/dts/overlays/w1-gpio-pullup-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/";
--- a/arch/arm/boot/dts/overlays/wittypi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/wittypi-overlay.dts
@@ -8,7 +8,7 @@
 
 / {
 
-	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&leds>;
